Config/AltBuildSystems/TasmanianSG.py

```diff
@@ -644,6 +646,74 @@ def makeWaveletGrid(self, iDimension, iOutputs, iDepth, iOrder=1, liLevelLimits=
 
         self.pLibTSG.tsgMakeWaveletGrid(self.pGrid, iDimension, iOutputs, iDepth, iOrder, pLevelLimits)
 
+    def makeFourierGrid(self, iDimension, iOutputs, iDepth, sType, liAnisotropicWeights=[], liLevelLimits=[]):
+        '''
+        creates a new sparse grid using a Fourier rule
+        discards any existing grid held by this class
+
+        iDimension: int (positive)
+              the number of inputs
+
+        iOutputs: int (non-negative)
+              the number of outputs
+
+        iDepth: int (non-negative)
+                controls the density of the grid, i.e.,
+                the offset for the tensor selection, the meaning of
+                iDepth depends on sType
+                Example 1: sType == 'iptotal' will give a grid that
+                           interpolates exactly all polynomials of
+                           degree up to and including iDepth
```

Comment:
* this relates to `OneDimensionalMeta::getIExact()` and `OneDimensionalMeta::getQExact()`, which currently default Fourier grids to just `level`
* remove the two examples from the comment, we will make examples later once we figure out the frequencies
* you have to change the main file and then regenerate the windows/GNU Make files